web/src/message_scroll.js

```diff
@@ -127,7 +134,7 @@ export function initialize() {
         "scroll",
         _.throttle(() => {
             unread_ops.process_visible();
-            message_lists.current.view.update_sticky_recipient_headers();
+            message_lists.current?.view.update_sticky_recipient_headers();
             scroll_finish();
         }, 50),
     );
```

Comment:
Hmm, this logic might deserve some follow-up attention; do we want to call any of these other functions when scrolling in inbox or some other view? I guess I'm not sure if those would get processed as scrolls on document.

Reply:
Yeah, added a commit for this.